warehousing-alternatives-compare-table.md

```diff
@@ -4,15 +4,15 @@
 
 technology | available document extract endpoints | notes
 --- | --- | --- 
-Apache Bahir Spark connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
+Apache Bahir Spark `sql-cloudant` connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
 IBM Cloud Functions | `/{db}/_changes`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | Cloud Function trigger is only available for `_changes` endpoint.
 Node-RED | `/{db}/_design/{design_doc}/_search/{search_index_name}` | -
 
 ### Sink options to Db2:
 
 technology | available SQL operations for writing to Db2 | notes
 --- | --- | ---
-Apache Bahir Spark connector | `CREATE TABLE`, `INSERT` | Spark save modes change table creation behavior:<br/>`SaveMode.ErrorIfExists` - try to create the table and error if it exists.<br/>`SaveMode.Append` - try to create the table, insert rows into existing table if present.<br/>`SaveMode.Overwrite` - try to create the table, dropping it first if it already exists.<br/>`SaveMode.Ignore` - try to create the table, doing nothing if it already exists.
+Apache Bahir Spark `sql-cloudant` connector | `CREATE TABLE`, `INSERT` | Spark save modes change table creation behavior:<br/>`SaveMode.ErrorIfExists` - try to create the table and error if it exists.<br/>`SaveMode.Append` - try to create the table, insert rows into existing table if present.<br/>`SaveMode.Overwrite` - try to create the table, dropping it first if it already exists.<br/>`SaveMode.Ignore` - try to create the table, doing nothing if it already exists.
```

Comment:
Isn't the save part independent of what loaded the data into the dataframe?

Reply:
Yes.  Since this functionality just requires a dataframe (regardless of what data is in it), it's probably better to just say "Apache Spark".